### PR TITLE
activitypub.md: slight language correction

### PR DESCRIPTION
**Pull request by orblivion**

### CODE REVIEW

**Comment by trwnh** [comment]
this change actually makes the sentence grammatically incorrect

**Comment by orblivion** [comment]
Upon reflection, you're likely correct that my word choice is wrong. My thinking was: sometimes people say "as best as we can" when they should (I'm pretty sure) say "as best we can" or "as well as we can". I saw "as best as possible" and made the equivalent of the first transformation. However it's wrong, or at very least awfully hard to parse.

However I'm still pretty sure "as best as possible" is also incorrect. If "best" means "most good", this translates to "as most good as possible" which doesn't make sense.

Ideas: "As well as possible" - though it may not have the same tone. "In the best way possible" - though it might be slightly confusing in this context. "As faithfully as possible"? "In as useful a way a possible"? The author may need to pick something that conveys the intended message.
